amplitude/CHANGELOG.md

```diff
@@ -0,0 +1,120 @@
+3.0.2 / 2019-09-16
```

Comment:
Would be possible to structure the monorepo as:
```
.circleci/
.github/
integrations/amplitude/
sampleapp/ <-- this later
README.md
LICENSE.md
etc.
```

Reply:
Okay, I actually do think this is a better structure. I'm on it